file/file.txt

```diff
@@ -0,0 +1,46 @@
+As its name states, EETS was begun as a 'club', and it retains certain features of that even now. 
```

Comment:
It's a bad name for the file and file location.
It's not clear from the file name what it is and why do we need it.

Reply:
Fixed

Reply:
It's not fixed. This file doesn't contain documentation.

Reply:
file renamed